focus.el

```diff
@@ -195,22 +195,21 @@ default is overwritten. This function simply helps set the
     (setq focus-current-thing (intern thing))))
 
 (defun focus-pin ()
-  "Pin the focused section to its current location or the region,
-if active."
+  "Pin the focused section to its current location or the region, if active."
   (interactive)
-  (when focus-mode
+  (when (bound-and-true-p focus-mode)
```

Comment:
I am not quite sure about this operation. I tested the package before I made this PR, but better if you test it again. :D